### PR TITLE
add back gql gen code + remove deploy script

### PR DESCRIPTION
**Pull request by avimoondra**
**TL;DR... We decided not checking in types is not worth the hassle!**  (or rather... checking them in is worth the convenience w/minimal impact to development)

The benefit to not checking in types is no merge conflicts!

To not check in types, the workflow we needed was...
- Generate types in GH Actions, for app and actions-server -- which depends on Hasura and Postgres to be running.
- Zip up a build with code, generated types, and installed packages, and upload as an artifact.
- From GH Actions, trigger a deploy via deploy hook to render.com.
- Have a deploy script in render services, that pull down the latest artifacts uploaded, unpack them and start the service.

Related PRs...
- https://github.com/avimoondra/fishbowl/pull/88
- https://github.com/avimoondra/fishbowl/pull/92 (and https://github.com/avimoondra/fishbowl/commit/afcc5898c9167376d55259a91e279f7f45f751fd)

Some issues that came up with this approach...
- a complex first-time set up - https://github.com/avimoondra/fishbowl/pull/127
- a bug where render was deploying previous SHA's artifacts, because there was a race condition in render starting the deploy process and the latest artifacts being available. i.e. "Artifacts allow you to share data between jobs in a workflow and store data **once that workflow has completed**." [see link](https://docs.github.com/en/actions/guides/storing-workflow-data-as-artifacts#:~:text=Artifacts%20allow%20you%20to%20share,Enterprise%20Server%2C%20and%20GitHub%20One.). We were notifying Render before completing the workflow. The proper fix for this would be to tag the artifact with the SHA, and have the deploy script poll for artifacts with that SHA.

Instead we'll simplify our deploy process by checking in types for app and actions-server, and then using render.com's "deploy on merge to master" setting. This means GH Actions is no longer blocking, and we can deploy code that is not passing formatter, linter or type checks. But the surface area for interruption is low since we have checks on PRs + Render will fail to deploy a bad build.

## Github changes

We can remove these secrets, defined here: https://github.com/avimoondra/fishbowl/settings/secrets/actions

image

## Render.com changes

fishbowl-app
- remove GITHUB_TOKEN
- change build command to: `cd app; yarn install --frozen-lockfile; yarn build`. Previously was `cd deploy; yarn install --frozen-lockfile; yarn run start app; mkdir ../app/build; unzip app.zip -d ../app/build`.
- turn on Auto Deploy

fishbowl-actions
- remove GITHUB_TOKEN
- change build command to: `cd actions-server; yarn install --frozen-lockfile; yarn build`. Previously was `cd deploy; yarn install --frozen-lockfile; yarn run start actions-server; unzip actions-server.zip -d ../actions-server/build; cd ../actions-server; yarn install --frozen-lockfile`.
- turn on Auto Deploy

### CODE REVIEW

**Comment by avimoondra** [comment]
@namoscato spot check me on this one! There may be some other areas of the codebase to update

**Comment by avimoondra** [comment]
[Deploy bug - deploying the action-server didn't deploy code from latest commit/build.](https://trello.com/c/jRUmW2Pl/113-deploy-bug-deploying-the-action-server-didnt-deploy-code-from-latest-commit-build)

**Review by namoscato — APPROVED** [review]
I think you hit everything, @avimoondra! I just made a few minor updates in 78c267a90323e0d8ba1aedfeb22e67541ddb05a5.

I'm glad we're simplifying this 😅

File: README.md [hunk 1]
```diff
@@ -100,10 +100,21 @@ yarn gql-gen --watch
...
+```bash
+docker-compose exec [fishbowl-app|fishbowl-actions-server] yarn [add|remove|etc.] xyz-package
+```
+
+Once the lockfile has been updated, you'll likely want to reinstall dependencies on your host to facilitate IDE autocompletion:
 
 ```bash
-docker-compose exec [actions-server|app] yarn [add|remove|etc.] xyz-package
+cd [actions-server|app]
+yarn install --frozen-lockfile
```
Comment (avimoondra) [inline README.md]: interesting, did not realize this. thanks
